test/com/google/javascript/jscomp/PeepholeSubstituteAlternateSyntaxTest.java

```diff
@@ -335,7 +335,8 @@ public void testUndefined() {
     foldSame("try {} catch(undefined) {}");
     foldSame("for (undefined in {}) {}");
     foldSame("undefined++;");
-    fold("undefined += undefined;", "undefined += void 0;");
+    disableNormalize();
+    foldSame("undefined += undefined;");
```

Comment:
https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/PeepholeSubstituteAlternateSyntax.java#L326
`isASTNormalized()` is now false in this case.